recipes/transposonpsi/build.sh

```diff
@@ -0,0 +1,22 @@
+TPSI_DIR=${PREFIX}/share/transposonPSI
+TPSI_PROGRAMS="BPbtab TBLASTN_hit_chainer.pl TBLASTN_hit_chainer_nonoverlapping_genome_DP_extraction.pl TPSI_btab_to_gff3.pl TPSI_chains_to_fasta.pl TPSI_chains_to_gff3.pl m2fmt_tier_hits.pl transposon_db_m2fmt_to_gff3.pl"
+
+# Copy the installed package to the right location
+mkdir -p ${PREFIX}/bin
+mkdir -p ${TPSI_DIR}
+cp -r *  ${TPSI_DIR}
+
+
+cat <<END >>${PREFIX}/bin/transposonPSI.pl
+#!/bin/bash
```

Comment:
Wouldn't a symlink be easier?

Reply:
I gave a shot but It did not work because FindBin looks for perl libs and other dependencies that must be in the same place as the link. I didn't want to mess the $PREFIX/bin it is why I copy everything in share and fake the presence of the scripts in the bien that are actually executed from  $PREFIX/bin/transposonpsi/